src/phoenix/core/model_schema_adapter.py

```diff
@@ -137,3 +150,37 @@ def _translate_embedding(
         link_to_data=embedding.link_to_data_column_name,
         display_name=display_name,
     )
+
+
+def _guess_vectors_vs_scalars(
+    names: Iterable[str],
+    *dataframes: pd.DataFrame,
```

Comment:
why the plural here when it's a single dataframe?

Reply:
the `*` marks it as variadic so it can be multiple